lib/mayaUsd/render/mayaToHydra/renderOverrideUtils.h

```diff
@@ -66,13 +66,17 @@ class HdMayaManipulatorRender : public MHWRender::MSceneRender {
         : MHWRender::MSceneRender(name) {}
 
     MUint64 getObjectTypeExclusions() override {
-        return ~MHWRender::MFrameContext::kExcludeManipulators;
+        return MHWRender::MSceneRender::getObjectTypeExclusions() | ~(kExcludeManipulators | kExcludeHoldOuts);
```

Comment:
Is this change relevant to the issue? Why do you exclude holdouts?

`MHWRender::MSceneRender::getObjectTypeExclusions()` returns 0, so there is no need to use it in the bitmask op.